tests/unit/src/test/scala/tests/troubleshoot/ProblemResolverSuite.scala

```diff
@@ -124,21 +122,6 @@ class ProblemResolverSuite extends FunSuite {
     scalacOpts = List("-Xplugin:/semanticdb-scalac_2.12.12-4.4.2.jar"),
   )
 
-  checkRecommendation(
```

Comment:
The test no longer makes sense, since we have fallbacks to the current java home.